src/main/java/org/folio/inventory/validation/experimental/InProcessTargetValidator.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import static java.util.Set.of;
+
+public class InProcessTargetValidator extends AbstractTargetValidator {
+
```

Comment:
Please remove this redundant blank line, I don't think it improves the readability of the code